package.json

```diff
@@ -5,7 +5,7 @@
 	"homepage": "https://hollama.fernando.is",
 	"description": "A minimal UI for talking to Ollama servers",
 	"scripts": {
-		"dev": "vite dev --host",
+		"dev": "concurrently \"typesafe-i18n\" \"vite dev --host\"",
```

Comment:
`typesafe-i18n` has a "watcher" process that listens to changes in the locales and re-generates the type definitions, so I added the `concurrently` library to run both processes at the same time when running `npm run dev`.